silo-core/contracts/utils/liquidationHelper/LiquidationHelper.sol

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.28;
+
+import {Address} from "openzeppelin5/utils/Address.sol";
+import {IERC20} from "openzeppelin5/token/ERC20/IERC20.sol";
+
+import {IERC3156FlashBorrower} from "../../interfaces/IERC3156FlashBorrower.sol";
+import {IPartialLiquidation} from "../../interfaces/IPartialLiquidation.sol";
+import {ILiquidationHelper} from "../../interfaces/ILiquidationHelper.sol";
+
+import {ISilo} from "../../interfaces/ISilo.sol";
+import {ISiloConfig} from "../../interfaces/ISiloConfig.sol";
+import {IWrappedNativeToken} from "../../interfaces/IWrappedNativeToken.sol";
+
+import "./DexSwap.sol";
+
+interface ISiloConfigHelper {
+    function siloConfig() external view returns (ISiloConfig);
+}
```

Comment:
Do we need it?
```suggestion
```

Reply:
I was using it before but code was removed, and I forgot to remove this one.